charts/hny-network-agent/templates/NOTES.txt

```diff
@@ -0,0 +1,18 @@
+===============================================================================================================
+   __ __   _  __ __  __        _  __       __                       __          ___                     __ 
+  / // /  / |/ / \ \/ /       / |/ / ___  / /_ _    __ ___   ____  / /__       / _ |  ___ _ ___   ___  / /_
+ / _  /  /    /   \  /       /    / / -_)/ __/| |/|/ // _ \ / __/ /  '_/      / __ | / _ `// -_) / _ \/ __/
+/_//_/  /_/|_/    /_/       /_/|_/  \__/ \__/ |__,__/ \___//_/   /_/\_\      /_/ |_| \_, / \__/ /_//_/\__/ 
+                                                                                    /___/                  
+
+
+Honeycomb Network Agent is setup and configured to collect events from your Kubernetes cluster.
+You should see data flowing within a few minutes at https://ui.honeycomb.io.
+
+{{ if .Values.honeycomb.apiKey }}
```

Comment:
@TylerHelmuth I think we've lost our validation rules for checking the following conditions:
- Neither honeycomb.apiKey or honeycomb.existingSecret is set
- Both honeycomb.apiKey or honeycomb.existingSecret are set
- honeycomb.existingSecret is set and honeycomb.existingSecretKey is not set